app/questionnaire_state/question.py

```diff
@@ -1,6 +1,23 @@
+from app.questionnaire_state.answer import Answer
+from app.questionnaire_state.item import Item
 
 
-class Question(object):
+class Question(Item):
     def __init__(self, id):
-        self.id = id
+        super().__init__(id=id)
         self.answers = []
+        self.children = self.answers
+
+    def get_answers(self):
+        answers = []
+        for child in self.children:
+            answers.append(child.get_answer())
```

Comment:
Do you mean append or do you mean to use extend? get_answers seems to return a list even if a single value is found, so I think you end up with a list of lists rather than just a single list of answers. Also I can't see the difference between this method and the parent method on the Item class shown in https://github.com/ONSdigital/eq-survey-runner/pull/313/files#diff-5f2cdd4ed301f540243d3474c5f36f01R24